specs/jsonschema-core.md

```diff
@@ -569,11 +601,19 @@ the keyword's value. Alternatively, an applicator may refer to a schema
 elsewhere in the same schema document, or in a different one. The mechanism for
 identifying such referenced schemas is defined by the keyword.
 
-Applicator keywords also define how subschema or referenced schema boolean
-[assertion](#assertions) results are modified and/or combined to produce the
-boolean result of the applicator. Applicators may apply any boolean logic
-operation to the assertion results of subschemas, but MUST NOT introduce new
-assertion conditions of their own.
+Applicator keywords also behave as assertions, using the assertion results of
+each subschema or referenced schema of the keyword. These boolean results are
+modified (e.g. the `not` keyword negates its subschema's assertion) and/or
+combined (e.g. `allOf` takes the conjunction of its subschemas' assertions) to
+produce the boolean result of the applicator. Applicators may apply any boolean
+logic operation to the assertion results of subschemas, but SHOULD NOT introduce
+new assertion conditions of their own.[^2]
+
+[^2]: It is recommended that keywords have a single resposibility. An example of
+this in action is the separation between `properties`, which verifies object
+property values have the right data _if_ they exist, and `required`, which
+verifies that object properties exist. Separating these concerns allows for more
+reusable components.
```

Comment:
More a broader comment that comes to mind with this change...

I know we have at times talked about the posibility of adding "shortcut" keywords, such as combining `properties` and `required`. I forget if this is something we explicitly decided we would or would not do. (I would be happy stating we will not do such, but I wanted to see what those with a better memory remember also!)